packages/site/public/index.html

```diff
@@ -1,15 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8" />
     <title>Featurevisor Status</title>
-    <link rel="stylesheet" href="./index.css" />
     <link rel="icon" type="image/png" href="./favicon-128.png" />
+    <script src="https://cdn.tailwindcss.com/"></script>
```

Comment:
is it possible to avoid loading anything from a remote location?

Reply:
I believe we only need this in development mode. I'll take a look.

Reply:
This is fixed now 👌 Removed it entirely both for development and production mode.